recipe/meta.yaml

```diff
@@ -19,15 +19,19 @@ source:
     - patches/root-6.16.02_allow_setting_clang_version.patch  # [osx]
     - patches/root-x.xx.xx_python_mac_os.patch  # [osx]
     - patches/root-x.xx.xx-copy-abspath-headers.patch  # [osx]
+    - patches/root-x.xx.xx-osx-remove-hardcoded-sysroot.patch  # [osx]
```

Comment:
Should this been made to ROOT?

Reply:
I wasn't planning to but a CMake option could be added though I'm not sure it would ever be useful for anything other than conda.

Reply:
Okay. Might be worth a bit of discussion, but it's probably mot worth of an option. Most systems I can think of would have a static location of the current sdk and build separately for each version. Maybe gentoo prefix, but I think even that has per macOS version builds.